scripts/release_scripts/generate_release_updates.py

```diff
@@ -137,6 +141,28 @@ def prompt_user_to_send_announcement_email():
         'Ensure the email sent to oppia@ is in the Announcements category')
 
 
+def create_group_for_next_release():
+    """Asks the release co-ordinator to create a new chat group for
+    the next release.
+    """
+    common.open_new_tab_in_browser_if_possible(
+        'https://github.com/oppia/oppia/wiki/Release-Schedule#'
+        'release-coordinators-and-qa-coordinators-for-upcoming-releases')
+    common.ask_user_to_confirm(
+        'Please check the Release co-ordinator & QA Lead for the next '
```

Comment:
Reword to make the main action come first. (Also, I don't think you need me to be in it -- let's at least try that for a bit and see how that goes.)

```
Please do the following two things:
1. Create a new chat group for the next release, and add the release coordinator, QA lead, Ankita and Nithesh to that group. You can find the release schedule and coordinators here (this tab should have just opened): https://....
2. Please send the following message to ...
```

Reply:
Done.